IABot/checkIfDead.php

```diff
@@ -130,7 +110,7 @@ protected function processResult( $headers, $curlerrno, $url ) {
 			if ( $httpCode == 401 || $httpCode == 503 || $httpCode == 507 ) {
 				return false;
 			} else {
-				return true;
+				return $this->checkWithoutHeadRequest( $url );
```

Comment:
Please add a comment explaining why this is necessary (i.e. Some sites don't allow HEAD requests).